n3fit/src/n3fit/scripts/evolven3fit_new.py

```diff
@@ -0,0 +1,149 @@
+"""
+This module contains the CLI for evolven3fit_new
+"""
+import logging
+import pathlib
+from argparse import ArgumentParser
+import sys
+import numpy as np
+
+from evolven3fit_new import eko_utils, evolve, cli
+
+_logger = logging.getLogger(__name__)
+
+
+def construct_eko_parser(subparsers):
+    parser = subparsers.add_parser("produce_eko",
+        help="Produce the eko for the specified theory. The q_grid starts at the Q0 given by the theory but the last point is q_fin and its number of points can be specified by q_points. The x_grid starts at x_grid_ini and ends at 1.0 and contains the provided number of points. The eko will be dumped in the provided path.")
```

Comment:
Its probably sensible to use implicit string joining to avoid the long lines here. E.g. help=("One sentence. ""Another sentence.")